outlook/src/classes/Partner.ts

```diff
@@ -22,6 +22,7 @@ class Partner {
     created: boolean;
     leads?: Lead[];
     tickets?: HelpdeskTicket[];
+    isCompany: boolean;
```

Comment:
Maybe could make an additional commit to re-order some big properties list using an alphabetical order ? Just to have a quick cleaning pass as code has evolved quite a lot recently.

Reply:
A code cleaning pass is indeed needed.

Since some other not yet merged tasks are bringing some changes, wouldn't it better to do a cleaning pass in a task dedicated for that (which will be merged after merging the other tasks)?

The cleaning pass would go through the entire outlook plugin codebase and it will:

- clean brackets, and linting issues
- rearrange methods and properties
- clean the CSS
- eliminate the use of inline styles which are still used in some places

Can you tell me what do you think about it?

Reply:
Ok, will do once we closed all other tasks, seems legit :)